### PR TITLE
Feature/generate random handles

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

File: app/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -26,6 +26,10 @@ public function update(Request $request)
                 'required',
                 (new Unique('users'))->ignore($user->id),
             ],
+            'handle' => [
+                'required',
+                (new Unique('users'))->ignore($user->id),
```
Comment (m1guelpf) [inline app/Http/Controllers/UsersController.php]: I normally use the app helper to make this cleaner... does that match your view or you prefer plain old object instantiation?

Comment (assertchris) [reply app/Http/Controllers/UsersController.php]: What does the app helper look like?

Comment (m1guelpf) [reply app/Http/Controllers/UsersController.php]: Forget it, just realized you wouldn't be able to pass the `users` parameter

**Comment by assertchris** [comment]
Changed to using the form (recommended in the docs](https://laravel.com/docs/5.6/validation#rule-unique)

**Pull request merged**